content/posts/open_channel_pre_taproot.md

```diff
@@ -0,0 +1,280 @@
+---
+title: "From `open_channel` to `channel_announcement`"
+summary: "A deep dive into opening a Lightning Network channel pre taproot"
+date: 2023-01-17
+aliases:
+  - /open_channel_pre_taproot
+---
+
+In this article, I will go over the current process of opening a Lightning
+Network channel. I will start with the `channel_open` message and will talk in
+detail about it and all the messages involved in “opening” the channel ending
+with the `channel_announcement` message. 
+
+The point of this article is to recap the current channel open flow so that a
+follow article describing the opening of a taproot
+channel[^taproot-channel-proposal] can focus only on the changes that taproot
+channels bring to the process.
+
+I previously wrote a [short article on channel opens] but I plan on revamping it
+here and diving into it in way more detail and with more diagrams (yay
+diagrams!). I will assume that the reader understands the structure of
+a commitment transaction and understands why they are asymmetric. If you feel
+you are a bit rusty in this area, please check out my articles explaining these
+[concepts in detail].
+
+## The end goal
+
+Before we dig into the first step, I think it is useful to illustrate the goal
+we are trying to reach. I will once again use our trusty peers, Alice and Bob,
+to tell the story. 
+
+Alice and Bob are both Lightning Network nodes. Alice’s node ID (ie, the public
+key used to identify her node) is `alice_node_id` and Bob’s node ID is
+`bob_node_id`. Their main two goals are: 
```

Comment:
main two -> two main:
https://www.gingersoftware.com/content/grammar-rules/adjectives/order-of-adjectives/

Reply:
🤓